deploy/helm/grafana-operator/values.yaml

```diff
@@ -143,6 +143,9 @@ securityContext:
   # -- Whether to require a container to run as a non-root user
   runAsNonRoot: true
 
+# -- Enable/Disable user namespaces for pods on Kubernetes version 1.33 and above
```

Comment:
I think the comment here is confusing due to the flipped logic of the setting itself. - To opt-in, it has to be set to `false` whereas the comment would make me think it's vice versa. 

I think it would better be set to something around the way how it's described in docs, e.g.:
`# -- Set to false to opt-in to use user namespaces`

As a side note, not all runtimes support the feature (e.g. containerd in AKS with `Ubuntu` SKU does not), so I guess it's better to keep it to `true` for now like you suggested here.